pending/adobe_common.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2015 Andrew Ziem
+    http://bleachbit.sourceforge.net
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	@app     Adobe Common
+	@os      Windows
+	@version 0.3.0
+-->
+<cleaner id="adobe_common" os="windows">
+  <label>Adobe</label>
+  <description>Shared/Common Adobe Applications</description>
+  <option id="logs">
+    <label>Logs</label>
+    <description>Delete the logs</description>
+    <!-- Application Manager Updater -->
+    <action command="delete" search="glob" path="$LOCALAPPDATA\Adobe\AAMUpdater\*\*.log"/>
+	<!-- Cloud Sync Library -->
+    <action command="delete" search="glob" path="$APPDATA\Adobe\ACSL\logs\*.txt"/>
+    <!-- Service Manager -->
+    <action command="delete" search="glob" path="$APPDATA\Adobe\CS*ServiceManager\logs\*.log"/>
+  </option>
+  <option id="cache">
+    <label>Cache</label>
+    <description>Delete the cache</description>
+    <!-- Service Manager -->
+    <action command="delete" search="glob" path="$APPDATA\Adobe\CS*ServiceManager\cache\*.*"/>
+    <!-- Camera Raw Library -->
+    <action command="delete" search="glob" path="$LOCALAPPDATA\Adobe\CameraRaw\Cache\*.dat"/>
+	<!-- Media Caches -->
+	<action command="delete" search="glob" path="$APPDATA\Adobe\Common\Media Cache Files\*"/>
+    <action command="delete" search="glob" path="$APPDATA\Adobe\Common\Media Cache\*"/>
+  </option>
+  <option id="installers">
+    <label>Installers</label>
+    <description>Delete cached patches and installers</description>
+    <action command="delete" search="glob" path="$PROGRAMFILES\Adobe\Adobe\AdobePatchFiles\*.*"/>
+    <action command="delete" search="glob" path="$PROGRAMFILES(X86)\Adobe\Adobe\AdobePatchFiles\*.*"/>
```

Comment:
`$PROGRAMFILES` -> C:\Program Files (x86) :boom:
`$PROGRAMFILES(X86)` -> :boom::boom::boom:

On my Windows 8.1 x64 the environment variables are misinterpreted!